Updated the generated python code

ranvijaykumar opened the pull request:
Updated the generated python code as per the latest version of the product. Also corrected the name of the dprep file.

azurecla commented:
Hi __@sebastienros__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sebros). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ranvijaykumar committed:
Updated the generated python code

Updated the generated python code as per the latest version of the product. Also corrected the name of the dprep file.
